gradio/components.py

```diff
@@ -652,6 +655,26 @@ def __init__(
             **kwargs,
         )
 
+    @staticmethod
+    def round_to_precision(num: float | int | None, precision: int | None):
+        """Round to a given precision.
+        If precision is None, no rounding happens. If 0, num is converted to int.
+
+        Parameters:
+        num (float | int): Number to round.
+        precision (Optional[int]): Precision to round to.
```

Comment:
Let's have definition and docstring the same
```suggestion
        precision (int | None): Precision to round to.
```